variables.tf

```diff
@@ -223,6 +233,11 @@ variable "deregistration_delay" {
   default     = 20
 }
 
+variable "listeners" {
+  description = "Optional; The ALB listener configuraiton."
+  default     = [{ port = 80, protocol = "HTTP", action = { type = "forward" } }]
```

Comment:
Default should be 80 redirects to 443, right?

Reply:
Oh, nevermind.  We only did that before if a cert was passed.  I would _love_ to keep that functionality to handle redirects if cert was passed.

What if we defaulted to empty array, and then if an empty array was passed, check for cert, and append 80-443-redir and 443-fwd, otherwise just 80-fwd?